pkg/apis/cloudcore/v1alpha1/default.go

```diff
@@ -0,0 +1,168 @@
+					ServiceEvent:               constants.DefaultServiceEventBuffer,
+					EndpointsEvent:             constants.DefaultEndpointsEventBuffer,
+					QueryPersistentvolume:      constants.DefaultQueryPersistentVolumeBuffer,
+					QueryPersistentvolumeclaim: constants.DefaultQueryPersistentVolumeClaimBuffer,
+					QueryVolumeattachment:      constants.DefaultQueryVolumeAttachmentBuffer,
+					QueryNode:                  constants.DefaultQueryNodeBuffer,
+					UpdateNode:                 constants.DefaultUpdateNodeBuffer,
+				},
+				Context: EdgeControllerContext{
+					SendModule:     metaconfig.ModuleNameCloudHub,
+					ReceiveModule:  metaconfig.ModuleNameEdgeController,
+					ResponseModule: metaconfig.ModuleNameCloudHub,
+				},
+				Load: EdgeControllerLoad{
+					UpdatePodStatusWorkers:            constants.DefaultUpdatePodStatusWorkers,
+					UpdateNodeStatusWorkers:           constants.DefaultUpdateNodeStatusWorkers,
+					QueryConfigmapWorkers:             constants.DefaultQueryConfigMapWorkers,
+					QuerySecretWorkers:                constants.DefaultQuerySecretWorkers,
+					QueryServiceWorkers:               constants.DefaultQueryServiceWorkers,
+					QueryEndpointsWorkers:             constants.DefaultQueryEndpointsWorkers,
+					QueryPersistentvolumeWorkers:      constants.DefaultQueryPersistentVolumeWorkers,
+					QueryPersistentvolumeclaimWorkers: constants.DefaultQueryPersistentVolumeClaimWorkers,
+					QueryVolumeattachmentWorkers:      constants.DefaultQueryVolumeAttachmentWorkers,
+					QueryNodeWorkers:                  constants.DefaultQueryNodeWorkers,
+					UpdateNodeWorkers:                 constants.DefaultUpdateNodeWorkers,
+				},
+			},
+			DeviceController: DeviceController{
+				Enable: true,
+				Context: DeviceControllerContext{
+					SendModule:     metaconfig.ModuleNameCloudHub,
+					ReceiveModule:  metaconfig.ModuleNameDeviceController,
+					ResponseModule: metaconfig.ModuleNameCloudHub,
+				},
+				Buffer: DeviceControllerBuffer{
+					UpdateDeviceStatus: constants.DefaultUpdateDeviceStatusBuffer,
+					DeviceEvent:        constants.DefaultDeviceEventBuffer,
+					DeviceModelEvent:   constants.DefaultDeviceModelEventBuffer,
+				},
+				Load: DeviceControllerLoad{
+					UpdateDeviceStatusWorkers: constants.DefaultUpdateDeviceStatusWorkers,
+				},
+			},
+		},
+	}
+}
+
+// NewCommonCloudCoreConfig return a common CloudCoreConfig object
+func NewCommonCloudCoreConfig() *CloudCoreConfig {
+	return &CloudCoreConfig{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       Kind,
+			APIVersion: path.Join(GroupName, APIVersion),
+		},
+		KubeAPIConfig: KubeAPIConfig{
+			Master:     "",
+			KubeConfig: constants.DefaultKubeConfig,
+		},
+		Modules: Modules{
+			CloudHub: CloudHub{
+				Enable:            true,
+				NodeLimit:         10,
+				TLSCAFile:         constants.DefaultCAFile,
+				TLSCertFile:       constants.DefaultCertFile,
+				TLSPrivateKeyFile: constants.DefaultKeyFile,
+				Quic: CloudHubQuic{
```

Comment:
QUIC is not necessary for commonconfig, websocket is sufficient.

Reply:
done